packages/pipeline-editor/src/RuntimesWidget.tsx

```diff
@@ -34,15 +34,15 @@ export const KFP_SCHEMA = 'kfp';
  */
 class RuntimesDisplay extends MetadataDisplay<IMetadataDisplayProps> {
   renderExpandableContent(metadata: IDictionary<any>): JSX.Element {
+    const apiEndpoint = metadata.metadata.api_endpoint.endsWith('/')
+      ? metadata.metadata.api_endpoint
+      : metadata.metadata.api_endpoint + '/';
```

Comment:
thanks for the fix, should we do the same with the cos_endpoint?

Reply:
cos endpoint works ok, this is probably a behavior from kfp ui